HISTORY.md

```diff
@@ -1,6 +1,11 @@
 # FRocksdb Change Log
 
-# 5.17.2-artisans (07/02/2019)
+# 5.17.2-artisans-1.1 (11/26/2019)
+### Java API Changes
+* Allow rocks java to explicitly create `WriteBufferManager`.
+* Punch through more flags for rocks java's `BlockBasedTableConfig`.
```

Comment:
I would write it as:
`Add more flags for BlockBasedTableConfig, mostly around caching index + filter blocks and partitioned filters`
It would also nice to briefly mention for what we need these changes in Flink